### PR TITLE
Add QuickJSON qlgenerator v1.0.0

### PR DESCRIPTION
**Pull request by johan**
New cask: [QuickJSON](https://github.com/johan/QuickJSON) v1.0.0.

### CODE REVIEW

File: Casks/quickjson.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  version '1.0.0'
+  sha256 '874fc0afd431f68527929a0592c23a81b70d53e6032419e540551fccd662cfd2'
+
+  url "https://github.com/johan/QuickJSON/releases/download/v#{version}/QuickJSON.qlgenerator.zip"
+  appcast 'https://github.com/johan/QuickJSON/releases.atom'
+  name 'QuickJSON'
+  homepage 'https://github.com/johan/QuickJSON'
+  license :mit
+
+  preflight do
```
Comment (vitorgalvao) [inline Casks/quickjson.rb]: Why?

Comment (johan) [inline Casks/quickjson.rb]: I mimicked another working quicklook plugin, https://github.com/caskroom/homebrew-cask/blob/master/Casks/qlmarkdown.rb#L15-L17

This being my first cask, if you have a different suggestion, what would it be – deleting all three lines?

Comment (vitorgalvao) [inline Casks/quickjson.rb]: > This being my first cask, if you have a different suggestion, what would it be – deleting all three lines?

Yes, we handle unpacking stuff. That cask is clearly not the best example to go by, [has it has a reson to need that](https://github.com/caskroom/homebrew-cask/blob/79dfb2776bbfb490a07ea033153921152bd746d0/Casks/qlmarkdown.rb#L11#L13).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a97bd495c0af9fa466fee394e445cfaa4148c595. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by johan** [comment]
Thanks! :-D (Sorry I didn't get back to this sooner myself.)

**Pull request closed without merging**